models/cfscript/Functions.cfc

```diff
@@ -27,7 +27,12 @@ component {
             }
 
             var formattedText = words.toList(' ');
-            var rootSettingKey = token[2].last() == 'entity.name.function.cfml' ? 'function_declaration' : 'function_anonymous';
+            var rootSettingKey = token[2].last() == 'entity.name.function.cfml' ? 'function_declaration' : '    function_anonymous';
```

Comment:
It looks like there is extra white space in the setting key here?

Reply:
Removed excess spacing.